BUILD.md

```diff
@@ -338,6 +338,21 @@ from gfxreconstruct's root source directory. Then install with `make install`.
   - The [Android Platform tools](https://developer.android.com/studio/releases/platform-tools) for your specific platform
   - [Android SDK 26 (8.0 Oreo) or newer](https://guides.codepath.com/android/installing-android-sdk-tools)
   - [Android NDK 21.3.6528147 (r21d)](https://developer.android.com/ndk/guides/)
+- [Java JDK 1.11](https://jdk.java.net/11)
+
+#### Additional Linux Command-Linux Prerequisites
+
+Additional requirements for building from the Linux command-line:
+- Define `ANDROID_HOME`=(location SDKs are installed on your system by Android Studio)
+  - Refer to Android Studio to find out where the files are installed
+  - For example:<br/>
```

Comment:
Do we embed HTML in our Markdown @bradgrantham ?

Reply:
Removed the imbedded html